phpstan.neon

```diff
@@ -3,8 +3,9 @@ includes:
 
 parameters:
     level: 6
```

Comment:
We could try to reach 8 at some point maybe

Reply:
Adjusting it to level 7 results in 37 errors, most of them in our commands and codegen parts.